README.md

```diff
@@ -39,55 +73,206 @@ tools (such as `ceph-installer` and `gdeploy`):
 * `ceph-installer`
 * `gluster-gdeploy-copr`
 
-See sample ansible playbook `site.yml.sample` to check how it fits together.
+Playbook files:
 
-## Basic setup
+* `prechecks.yml`: playbook checking requirements before Tendrl installation
+  (see comments inside the playbook file for references)
+* `site.yml`: main playbook of tendrl-ansible, which one will use to install
+  Tendrl
 
-Ansible Driven installation:
 
-1) Install ansible >= 2.3
-2) Get the code: `git clone https://github.com/Tendrl/tendrl-ansible.git`
-3) Create Ansible inventory file with groups for `tendrl-server`
-   and `gluster-servers`. Here is an example of inventory
-   file for 4 node cluster with Gluster:
+## Where are the roles and playbooks if I use rpm package?
 
-```
-[gluster-servers]
-gl1.example.com
-gl2.example.com
-gl3.example.com
-gl4.example.com
-
-[tendrl-server]
-tendrl.example.com
-```
+Ansible roles are available in `/usr/share/ansible/roles/` directory, where
+the role directories are prefixed with `tendrl-ansible.`, for example:
+`/usr/share/ansible/roles/tendrl-ansible.tendrl-server`.
+Each role has it's own `README.md` file, where you can find all details about
+it's usage.
+
+Playbooks are available in `/usr/share/doc/tendrl-ansible-1.6.0/` directory,
+where `1.6.0` is version of tendrl-ansible package.
+
+
+## What should I know before using tendrl-ansible?
+
+You need to know [how to use
+ansbile](https://docs.ansible.com/ansible/latest/intro.html) and how to deploy
+and use ssh public keys.
+
+Moreover since this README file can't provide all details about Tendrl, you
+should read [Tendrl installation
+documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-release-latest)
+as well.
+
+And last but not least, both `tendrl-server` and `tendrl-storage-node` contains
+many variables which one can use to tweak the installation. See README files of
+the roles for their description.
+
+
+## What installation steps from Tendrl installation documentation are not part of tendrl-ansible?
+
+This is should be clear from [Tendrl installation
```

Comment:
Probably spare *is*... in the *This **is** should be clear...* sentence.

Reply:
fixed in a3091aa3f21b244c5d83fd5ec2dc8edcaf3e12c3